tests/unit/components/playback/controls/SettingsFormVuex.spec.js

```diff
@@ -0,0 +1,203 @@
+    expect(
+      wrapper.find("#input-dropdown-widget-user-").element.value
+    ).toStrictEqual("");
+  });
+  test("Given that the SettingsForm is loaded with Vuex store, When there is no data in the user_ids, Customer Index/User Index is not set in the Vuex, Then <empty> is set in the Input of  Customer ID/User ID", () => {
+    store.commit("settings/set_customer_account_ids", array_of_special_userids);
+    wrapper = mount(ComponentToTest, {
+      store,
+      localVue,
+    });
+    expect(
+      wrapper.find("#input-dropdown-widget-cust-").element.value
+    ).toStrictEqual("");
+    expect(
+      wrapper.find("#input-dropdown-widget-user-").element.value
+    ).toStrictEqual("");
+  });
+  test("Given that the SettingsForm is loaded with Vuex with different set of null user_ids from store, When there is no data in the user_ids Customer Index/User Index is not set in the Vuex, Then <empty> is set in the Input of  Customer ID/User ID", async () => {
+    store.commit(
+      "settings/set_customer_account_ids",
+      array_of_special_customerid_null_userids
+    );
+    store.commit("settings/set_customer_index", 0);
+    wrapper = mount(ComponentToTest, {
+      store,
+      localVue,
+    });
+
+    expect(
+      wrapper.find("#input-dropdown-widget-cust-").element.value
+    ).toStrictEqual("");
+    expect(
+      wrapper.find("#input-dropdown-widget-user-").element.value
+    ).toStrictEqual("");
+  });
+  test("Given that the SettingsForm is loaded with Vuex with 'Customer account -1' 'User account -1', When user decides to modify Customer ID to 'Customer account -2' 'Lab User -1', Then validate 'Save Changes' updates validate Vuex store with is updated index values", async () => {
+    store.commit("settings/set_customer_account_ids", array_of_customerids);
+    store.commit("settings/set_customer_index", 0);
+    store.commit("settings/set_user_index", 0);
+
+    wrapper = mount(ComponentToTest, {
+      store,
+      localVue,
+    });
+    await wrapper.vm.$nextTick(); // wait for update
+
+    await wrapper
+      .find("#input-dropdown-widget-cust-")
+      .setValue("Customer account -2");
+    await wrapper.vm.$nextTick(); // wait for update
+    await wrapper.find("#input-dropdown-widget-user-").setValue("Lab User -1");
+    await wrapper.vm.$nextTick(); // wait for update
+
+    await wrapper.vm.$nextTick(); // wait for update
+
+    wrapper.vm.savechanges();
+
+    await wrapper.vm.$nextTick(); // wait for update
+
+    expect(store.getters["settings/customer_index"]).toStrictEqual(1); // this is the real data due to savechanges function Vuex stored data of customer_index
+    expect(store.getters["settings/user_index"]).toStrictEqual(0); // this is the real data due to savechanges function Vuex stored data of user_index
+  });
+  test("Given that the SettingsForm is loaded with Vuex, When the 'Key Icon' decoder is invoked, Then validate if the decoder string converts UNICODE value to key icon", async () => {
```

Comment:
it doesn't look like this `decoder` function is being used by the component at the moment, which makes it hard to test without accessing the `vm` directly.  Since it's not being used, but we do plan to try and implement it later, I would suggest commenting out this test and the `decoder` function in the component for the moment

Reply:
This results in codecov failure and CI Build fails so would like to retain for just codecoverage metrics and CI Build perspective.